setup.cfg

```diff
@@ -8,4 +8,4 @@ exclude = *migrations*,build*
 statistics = true
 application-import-names = groups
 import-order-style = google
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D203,D204
```

Comment:
Perhaps I'm being dim, but "Missing docstring in public package" doesn't really mean a lot to me. What it it really asking for here?